Description: Add cpu feature sgx1.
Author: Tim Wiederhake <twiederh@redhat.com>
Origin: upstream, https://gitlab.com/libvirt/libvirt/-/commit/5d9cabd0c7f67f4a7a4d21bb965ce1167ae02945
Bug-Ubuntu: https://launchpad.net/bugs/2028057
Forwarded: not-needed
Last-Update: 2023-11-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/cpu_map/x86_features.xml
+++ b/src/cpu_map/x86_features.xml
@@ -419,6 +419,10 @@
   </feature>
 
   <!-- cpuid level 0x00000012 ecx 0x0 features -->
+  <feature name='sgx1'>
+    <cpuid eax_in='0x12' ecx_in='0x00' eax='0x00000001'/>
+  </feature>
+
   <feature name='sgx-exinfo'>
     <cpuid eax_in='0x12' ecx_in='0x00' ebx='0x00000001'/>
   </feature>
